JobConfig/pileup/prolog.fcl

```diff
@@ -280,7 +301,8 @@ Pileup: {
     "keep mu2e::CaloShowerSteps_compressDetStepMCs_*_*",
     "keep mu2e::CrvSteps_compressDetStepMCs_*_*",
     "keep mu2e::SimParticlemv_compressDetStepMCs_*_*",
-    "keep mu2e::GenParticles_compressDetStepMCs_*_*" ]
+    "keep mu2e::GenParticles_compressDetStepMCs_*_*",
+    "keep mu2e::StepPointMCs_*_virtualdetector_*" ]
```

Comment:
might be best to comment this out for official produciton

Reply:
yes I agree this would enlarge too much the dataset size, by the way it would be nice to have the possibility to compare for both e+ and e-, the track momentum at tracker entrance obtained by KinKal surface intersection with the particle momentum obtained by the virtual detector located at the tracker entrance.

Reply:
My preference is that the default be correct for production.  If people want to modify it for their development work or other studies, they are free to do so.  There are ways to accomplish this that do not involve commenting things out.   I have some ideas if you want to talk about them,

Reply:
ok, just take this change away. We will find the way to insert the virtual detector information for generated e+e- in the gamma conversion  campaign if we consider it necessary.